static/js/blackbox.js

```diff
@@ -0,0 +1,113 @@
+// Set the dimensions of the canvas / graph
+        var margin = {
```

Comment:
It used to be part of a function, I think, then I removed the function and forgot to fix the formatting.